oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/Configuration.java

```diff
@@ -395,4 +395,9 @@
             description = "Integer value indicating the number of documents to check for garbage in each Full GC cycle." +
                     "The default value is " + DocumentNodeStoreService.DEFAULT_FGC_PROGRESS_SIZE)
     int fullGCProgressSize() default DocumentNodeStoreService.DEFAULT_FGC_PROGRESS_SIZE;
+
+    @AttributeDefinition(
+            name = "Invisible for discovery",
+            description = ""Boolean value indicating whether the instance should be discoverable by the cluster. The default value is " + DocumentNodeStoreService.DEFAULT_INVISIBLE_FOR_DISCOVERY)
```

Comment:
This double quote symbol is breaking the build now:

```suggestion
            description = "Boolean value indicating whether the instance should be discoverable by the cluster. The default value is " + DocumentNodeStoreService.DEFAULT_INVISIBLE_FOR_DISCOVERY)